pkg/debuginfo/store.go

```diff
@@ -198,6 +200,20 @@ func validateID(id string) error {
 	return nil
 }
 
+func (s *Store) find(ctx context.Context, key string) (bool, error) {
+	found := false
+	err := s.bucket.Iter(ctx, key, func(_ string) error {
```

Comment:
I'm happy for this to be a follow-up, but I expected that we also check the upstream debug info endpoint here.

Reply:
To be crystal clear here, by the “the upstream debug info endpoint” you meant to look up the debuginfod servers that have been specified? 

Or do you want to utilize existing expose `Exists` API? 

For the first one, we check it at access time:

https://github.com/parca-dev/parca/blob/36b893ae5eb7adfe0ee28c34020b4d64fe123ced/pkg/debuginfo/store.go#L252-L260

Reply:
I did indeed mean the first one, thanks for clarifying! Ignore my comment then!